tests/build_and_test_docker_locally.py

```diff
@@ -0,0 +1,293 @@
+import json
+import logging
+import os
+import shutil
+import subprocess
+from unittest import TestCase
+
+import yaml
+from ebi_eva_common_pyutils.command_utils import run_command_with_output
+from ebi_eva_common_pyutils.logger import logging_config
+
+from eva_sub_cli.validators.docker_validator import DockerValidator
+from tests.test_utils import create_mapping_file
+
+logger = logging_config.get_logger(__name__)
+
+
+class TestDockerValidator(TestCase):
```

Comment:
For the unit tests, I feel like building locally is more useful than the original [docker validator test](https://github.com/EBIvariation/eva-sub-cli/blob/4416015b4de91568e2c52ac36357c824f3ad53a9/tests/test_docker_validator.py) - should we just replace that one with this one instead? Or replace the container download step in that test with the local build step of this one?

Reply:
As discussed, leaving as it is for now